buildpack.md

```diff
@@ -314,7 +305,6 @@ These buildpacks must be compatible with the app.
 **GIVEN:**
 - An ordered list of buildpack groups resolved into buildpack implementations as described in [Order Resolution](#order-resolution)
 - A directory containing application source code
-- A shell, if needed,
```

Comment:
This line always confused me a little. Is it referring to the presence of a shell in the detect/build environment, or at runtime? I think it's fine to delete it, but I don't really know what it meant in the first place.